HH_custom_primary_key.md

```diff
@@ -0,0 +1,97 @@
+###Example: Create a model with custom primary key `name:string` instead of autogenerated `id:integer`
```

Comment:
Please remove the header. Readme.io puts one in for us, and a double header in that layout doesn't really work.